### DISCUSSION TITLE
Searching Projects for Contributes Specimens

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#4882] [General]
When searching for projects that contribute specimens, I get different results when logged in and as a public user.

Logged in search
![image](https://user-images.githubusercontent.com/5725767/80319199-712bde80-87cc-11ea-90ad-69f2a54501b7.png)

Logged in results
![image](https://user-images.githubusercontent.com/5725767/80319186-5e190e80-87cc-11ea-8164-d4ce73e677de.png)

Public User search
![image](https://user-images.githubusercontent.com/5725767/80319206-8274eb00-87cc-11ea-9139-a033af9781b7.png)

Public User results
![image](https://user-images.githubusercontent.com/5725767/80319827-e77e1000-87cf-11ea-973a-3d13c74bd55b.png)

I wasn't expecting there to be any difference, especially this big of a difference and it appears that the results when logged in only show projects related to collections for which I have some sort of editing access. Being logged in or not doesn't appear to affect the results of any of the other searches I performed in https://github.com/ArctosDB/PG-migration-testing/issues/126

**Comment by dustymc** [comment]
I'm not seeing the difference - what am I missing?

**Comment by Jegelewicz** [comment]
I pasted in the same results twice. When not logged in, this is what I get
![image](https://user-images.githubusercontent.com/5725767/80319807-d208e600-87cf-11ea-9646-bcfc7d80be4d.png)

**Comment by dustymc** [comment]
Thanks.

That's joining to specimens, so the results are going to depend on what specimens your user can see.

**Comment by campmlc** [comment]
Don't we have an issue somewhere about allowing operators to see but not
touch specimens outside their VPD access? This is obviously outside of PG
testing, but it would be valuable especially in this case.

**Comment by dustymc** [comment]
> it would be valuable

Or maybe infuriating, depending on goals....

This would be a great use for an API - just pull stuff as some user, pull stuff as some other user, do whatever you want with the merge.

**Comment by Jegelewicz** [comment]
> That's joining to specimens, so the results are going to depend on what specimens your user can see.

I get that, but aren't projects supposed to be collaborative? How can I collaborate if I can't see it? Is there any way to make the project visible and exclude specimens for which I don't have access?

**Comment by dustymc** [comment]
Projects are collaborative. Everyone can see them all. You simply can't locate them by specimens which don't exist for your user.
